Providers/Modules/Plugins/Common/plugin/oms_configuration.rb

```diff
@@ -144,6 +159,17 @@ def notify_blob_ods_endpoint
         @@NotifyBlobODSEndpoint
       end # getter notify_blob_ods_endpoint
 
+      def azure_resource_id
+        @@AzureResourceId
+      end
+
+      def omscloud_id
+        @@OmsCloudId
+      end
+
+      def get_vm_uuid
```

Comment:
Inconsistent naming : get_X or X, Shouldn't X be private if accessors are needed?